.changeset/thick-bananas-whisper.md

```diff
@@ -0,0 +1,10 @@
+---
+"ilib-loctool-webos-javascript": major
```

Comment:
major update means that the software is incompatible with the previous version, which I don't think is the case here. I think you meant a minor patch which means a new feature. Patch is for fixes to the existing features.

v<major>.<minor>.<patch>

Reply:
yeah, I also wasn't sure if a major version was the right choice. so, following your suggestion, I'm going to change it to a minor version update